ekncontent/tests/ekncontent/testDomain.js

```diff
@@ -63,6 +63,13 @@ describe('Domain', function () {
             });
         });
 
+        it('throws for an ID that is not valid', function (done) {
+            domain.get_object("ekn:///02463d24cb5690af2c8e898736ea8c80e0e77077/", null, function (domain, result) {
```

Comment:
Better to make a different, "more invalid" example of an invalid ID, since we were talking about possibly accepting the trailing slash.